server/src/controllers/organizations.controller.ts

```diff
@@ -28,7 +28,7 @@ export class OrganizationsController {
     private organizationsService: OrganizationsService,
     private authService: AuthService,
     private readonly configService: ConfigService
-  ) {}
+  ) { }
```

Comment:
is this change intended ?

Reply:
Not intended, but not a breaking change.  VScode must have done that automatically.  Removed the space though.